README.md

```diff
@@ -72,7 +72,7 @@ instance of the adage-server.
 
    If you would like to load different data files from your own ADAGE model,
    you can do so, provided that they are in the same format as the
-   corresponding files in the `data/` folder.
+   corresponding files in the `data/` folder described above.
```

Comment:
Minor quibble: in the above-referenced text (at line 56, which I can't comment on directly because it's too far up from this diff) the Latin name _Pseudomonas_ should be italicized. So the line should read:

```
    If you want to load the default _Pseudomonas_ data into the server database,
```